extensions/transform/org.eclipse.smarthome.transform.javascript/src/main/java/org/eclipse/smarthome/transform/javascript/internal/JavaScriptTransformationService.java

```diff
@@ -12,36 +12,42 @@
  */
 package org.eclipse.smarthome.transform.javascript.internal;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.InputStreamReader;
-import java.io.Reader;
-
-import javax.script.ScriptEngine;
-import javax.script.ScriptEngineManager;
+import javax.script.Bindings;
+import javax.script.CompiledScript;
 import javax.script.ScriptException;
 
-import org.apache.commons.io.IOUtils;
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
-import org.eclipse.smarthome.config.core.ConfigConstants;
 import org.eclipse.smarthome.core.transform.TransformationException;
 import org.eclipse.smarthome.core.transform.TransformationService;
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * The implementation of {@link TransformationService} which transforms the
  * input by Java Script.
  *
- * @author Pauli Anttila
+ * @author Pauli Anttila Initial contribution
+ * @author Thomas Kordelle pre compiled scripts
```

Comment:
Thank you, I assume we use a `-` between name and description in the other files. Can you check?